meilisearch/tests/index/test_index.py

```diff
@@ -43,21 +47,20 @@ def test_index_info_with_wrong_uid(self):
 
     def test_get_primary_key(self):
         """Tests an API call to get primary-key of an index in MeiliSearch"""
-        index = self.client.get_index(uid='movies_uid')
+        index = self.client.get_index(uid=self.index_uid)
         response = index.get_primary_key()
         assert response is None
 
     def test_update_index(self):
         """Tests an API call to update an index in MeiliSearch"""
-        index = self.client.get_index(uid='movies_uid')
+        index = self.client.get_index(uid=self.index_uid)
         response = index.update(primaryKey='objectID')
         assert isinstance(response, object)
         assert index.get_primary_key() == 'objectID'
 
-    @pytest.mark.run(order=-1)
     def test_delete_index(self):
         """Tests an API call to delete an index in MeiliSearch"""
-        index = self.client.get_index(uid="movies_uid")
+        index = self.client.get_index(uid=self.index_uid)
```

Comment:
Should check if index was deleted by doing a call on 
```
client.get_indexes()
```

Reply:
Done with 

```python
with pytest.raises(Exception):
            self.client.get_index(uid=self.index_uid).info()